test/issue223-2.js

```diff
@@ -0,0 +1,27 @@
+
+var tape = require('tape');
+var mjAPI = require("../lib/mj-single.js");
+var jsdom = require('jsdom').jsdom;
+
+tape('displayAlign:left in HTML output with mlabeledtr', function(t) {
+  t.plan(1);
+  mjAPI.config({MathJax: {displayIndent: '10em'}
+});
+  mjAPI.start();
+  var tex = '\\begin{equation} x \\tag{1} \\end{equation}';
```

Comment:
There is no need for the `\\begin{equation}` and `\\end{equation}` here; `x \\tag{1}` should be sufficient.